lib/token-assert.js

```diff
@@ -38,6 +38,21 @@ TokenAssert.prototype.whitespaceBetween = function(options) {
                 column: token.loc.end.column
             });
         }
+    } else if (options.hasOwnProperty('maximumSpaces')) {
```

Comment:
We should probably clarify that those are about lines. It will be confusing to have them and `maximumSpaces`. We can rename to `atMostSpaces` for consistency.